src/component/splits/mod.rs

```diff
@@ -466,6 +480,22 @@ impl Component {
                 "Current Split Gradient".into(),
                 self.settings.current_split_gradient.into(),
             ),
+            Field::new(
+                "Split Time Accuracy".into(),
+                self.settings.split_time_accuracy.into(),
+            ),
+            Field::new(
+                "Segment Time Accuracy".into(),
+                self.settings.segment_time_accuracy.into(),
+            ),
+            Field::new(
+                "Delta Time Accuracy".into(),
+                self.settings.delta_time_accuracy.into(),
+            ),
+            Field::new(
+                "Drop Decimals When More Than 1 Minute".into(),
```

Comment:
The original LS has these all in groups, so it's obvious what this refers to. Here we don't have any groups yet, so maybe somehow include that this refers to the delta.

Reply:
what do you think of `Drop Delta Decimals When Showing Minutes`? or a more direct change to `Drop Delta Decimals When More Than 1 Minute`?

Reply:
The former seems better.